uint16 base
uint16 elbow_v
uint16 wrist_v
uint16 wrist_h
uint16 claws
